group/all.go

```diff
@@ -34,10 +31,19 @@ func init() {
 // registered suites.
 var ErrUnknownSuite = errors.New("unknown suite")
 
-// Suite return
+// Suite looks up a suite by name.
 func Suite(name string) (kyber.Group, error) {
 	if s, ok := suites[strings.ToLower(name)]; ok {
 		return s, nil
 	}
 	return nil, ErrUnknownSuite
 }
+
+// MustSuite looks up a suite by name and panics if it is not found.
+func MustSuite(name string) kyber.Group {
+	s, err := Suite(name)
+	if err != nil {
+		panic("Suite " + name + " not found.")
```

Comment:
I try to avoid panic - isn't an error enough?

Reply:
By analogy to https://godoc.org/regexp#MustCompile
"It simplifies safe initialization of global variables holding compiled regular expressions."

Reply:
And where is it used?